site/en/docs/privacy-sandbox/topics/topic-classification/index.md

```diff
@@ -0,0 +1,225 @@
+
+## How the user's top five topics are selected
+
+The API returns one topic for each epoch, up to a maximum of three. If three are returned, this includes topics for the current epoch and the previous two.
+
+1. At the end of each epoch, the browser compiles a list of pages that meet the following criteria:
+    - The page was visited by the user during the epoch.
+    - The page includes code that calls `document.browsingTopics()`
+    - The API was enabled (for example, not blocked by the user or via a [response header](https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy)).
+2. The browser, on the user's device, uses the classifier model provided by the Topics API to map the hostname for each page to a list of topics.
+3. The browser accumulates the list of topics.
+4. The browser generates a list of the top five topics by frequency.
+
+The `document.browsingTopics()` method then returns a random topic from the top five for each epoch, with a 5% chance that any of these may be randomly chosen from the full taxonomy of topics. In Chrome, users would also be able to remove individual topics, or clear their browsing history to reduce the number of topics returned by the API. Users may also [opt out](#opt-out) of the API.
+
+View information about topics observed during the current epoch from the `chrome://topics-internals` page.
+
+
+## How the API decides which callers see which topics
+
+API callers only receive topics they've recently observed, and the topics for a user are refreshed once each epoch. That means the API provides a rolling window in which a given caller may receive certain topics.
+The table below outlines an example (though unrealistically small) of a hypothetical browsing history for a user during a single epoch, showing topics associated with the sites they've visited, and the API callers present on each site (the entities that call `document.browsingTopics()` in JavaScript code included on the site).
+
+<table>
+  <thead>
+  <tr>
+  <th style="text-align: left;"><strong>Site</strong></th>
+  <th style="text-align: left;"><strong>Topics</strong></th>
+  <th style="text-align: left;"><strong>API callers on site</strong></th>
+  </tr>
+  </thead>
+  <tbody>
+    <tr>
+    <td>yoga.example</td>
+    <td>Fitness</td>
+    <td>adtech1.example adtech2.example</td>
+    </tr>
+    <tr>
+    <td>knitting.example</td>
+    <td>Crafts</td>
+    <td>adtech1.example</td>
+    </tr>
+    <tr>
+    <td>hiking-holiday.example</td>
+    <td>Fitness, Travel & Transportation</td>
+    <td>adtech2.example</td>
+    </tr>
+    <tr>
+    <td>diy-clothing.example</td>
+    <td>Crafts, Fashion & Style</td>
+    <td>[none]</td>
+    </tr>
+  </tbody>
+</table>
+
+At the end of the epoch (currently proposed to be one week) the Topics API generates the browser's top topics for the week.
+- adtech1.example is now eligible to receive the "Fitness" and "Crafts" topics, since it observed them on yoga.example and also on knitting.example.
```

Comment:
```suggestion
At the end of the epoch (currently proposed to be one week) the Topics API generates the browser's top topics for the week.

- adtech1.example is now eligible to receive the "Fitness" and "Crafts" topics, since it observed them on yoga.example and also on knitting.example.
```

Reply:
Same goes for lists and paragraphs, always add the extra line space between them.